$OpenBSD$

Force pickup of jsapi.h from lang/spidermonkey which has js-1.8
and NOT from bundled, old js-1.7

--- src/mongo/scripting/engine_spidermonkey.h.orig	Tue Dec  4 17:49:32 2012
+++ src/mongo/scripting/engine_spidermonkey.h	Tue Dec  4 17:49:48 2012
@@ -29,7 +29,7 @@
 #define JS_PUBLIC_DATA(t)   t
 #endif
 
-#include <third_party/js-1.7/jsapi.h>
+#include <jsapi.h>
 
 // END inc hacking
 
